serviceUsage.html

```diff
@@ -272,6 +274,27 @@ <h2>
             </tr>
         </table>
 
+        <h5>generateAndPublish endpoint is provided with two options for Lookups in swagger page:</h5>
+        <table class="lookups">
+        <tr>
+            <th style="text-align: left;">Options</th>
+            <th style="text-align: left;">Default Value</th>
+            <th style="text-align: left;">Description</th>
+         </tr>
+        <tr>
+            <td><h6>failIfMultipleFound:</h6></td>
+            <td>False</td>
+            <td>If value is set to false then all upstream/downstream event ids of that particular event are fetched from configured event repository in generate  and adds them in links section of the target field.
```

Comment:
This is not the intended behaviour of this, if a query returns more than 1 response, this value says whether to build the event with all the found event Ids, or not. Please check this in generate also.